src/goto-programs/README.md

```diff
@@ -8,100 +8,59 @@
 \section goto-programs-overview Overview
 Goto programs are the intermediate representation of the CPROVER tool
 chain. They are language independent and similar to many of the compiler
-intermediate languages. Section \ref goto-programs describes the
-`goto_programt` and `goto_functionst` data structures in detail. However
-it useful to understand some of the basic concepts. Each function is a
-list of instructions, each of which has a type (one of 18 kinds of
+intermediate languages. Each function is a
+list of instructions, each of which has a type (one of 19 kinds of
 instruction), a code expression, a guard expression and potentially some
 targets for the next instruction. They are not natively in static
 single-assign (SSA) form. Transitions are nondeterministic (although in
 practise the guards on the transitions normally cover form a disjoint
 cover of all possibilities). Local variables have non-deterministic
-values if they are not initialised.  Variables and data within the
-program is commonly one of three types (parameterised by width):
-`unsignedbv_typet`, `signedbv_typet` and `floatbv_typet`, see
-`util/std_types.h` for more information. Goto programs can be serialised
+values if they are not initialised. Goto programs can be serialised
 in a binary (wrapped in ELF headers) format or in XML (see the various
 `_serialization` files).
 
-The `cbmc` option `–show-goto-programs` is often a good starting point
+The `cbmc` option `--show-goto-programs` is often a good starting point
 as it outputs goto-programs in a human readable form. However there are
 a few things to be aware of. Functions have an internal name (for
 example `c::f00`) and a ‘pretty name’ (for example `f00`) and which is
 used depends on whether it is internal or being presented to the user.
-The `main` method is the ‘logical’ main which is not necessarily the
-main method from the code. In the output `NONDET` is use to represent a
-nondeterministic assignment to a variable. Likewise `IF` as a beautified
+In the output `NONDET` is use to represent a nondeterministic assignment to a
+variable. Likewise `IF` as a beautified
```

Comment:
I know that's just re-formatting of prior text, but none of these two sentences fully parse for me.